packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -353,10 +409,9 @@ describe('BlocksEditor toolbar', () => {
 
     const headingsDropdown = screen.getByRole('combobox', { name: /Select a block/i });
 
-    // Set the selection to cover the entire content
+    // Set the selection to cover the second and third row
     Transforms.setSelection(baseEditor, {
-      anchor: Editor.start(baseEditor, []),
-      focus: Editor.end(baseEditor, []),
+      anchor: { path: [1, 0], offset: 0 },
```

Comment:
you're only setting the anchor and not focus so it looks like you're setting the selection to only the second row 🤔